test/acceptance/test/wego_install_tests.go

```diff
@@ -1,88 +1,106 @@
 // +build smoke acceptance
 
+/**
+* All tests related to 'wego install' will go into this file
+ */
+
 package acceptance
 
 import (
 	"fmt"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
 	"os/exec"
 )
 
-func VerifyControllersInCluster(session *gexec.Session) {
+var (
+	err       error
+	namespace string
+)
 
-	By(" Then I should see flux controllers present in the cluster", func() {
-		Eventually(string(session.Wait().Out.Contents())).Should(ContainSubstring("helm-controller"))
-		Eventually(string(session.Wait().Out.Contents())).Should(ContainSubstring("kustomize-controller"))
-		Eventually(string(session.Wait().Out.Contents())).Should(ContainSubstring("notification-controller"))
-		Eventually(string(session.Wait().Out.Contents())).Should(ContainSubstring("source-controller"))
-	})
-}
+func VerifyControllersInCluster(session *gexec.Session) {
 
-//Reseting namespace is an expensive operation, only use this when absolutely necessary
-func ResetNamespace(namespace string) {
-	By("And there's no previous wego installation", func() {
-		command := exec.Command("sh", "-c", fmt.Sprintf("%s install --namespace %s| kubectl --ignore-not-found=true delete -f -", WEGO_BIN_PATH, namespace))
+	By("And I wait for the controllers to get ready", func() {
+		command := exec.Command("sh", "-c", fmt.Sprintf("%s wait --for=condition=Ready -n %s --all pod", "kubectl", namespace))
 		session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
 		Expect(err).ShouldNot(HaveOccurred())
 		Eventually(session, INSTALL_RESET_TIMEOUT).Should(gexec.Exit())
```

Comment:
Should add a new variable constant with timeout and reset timeout doesn't make much sense in this case.